.github/workflows/greeting.yaml

```diff
@@ -0,0 +1,27 @@
+name: Greeting
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  greeting:
+    permissions:
+      pull-requests: write
+    name: Greeting comment upon PR from fork
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository
+
+    steps:
+      - name: Issue the greeting comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Hi @${{ github.event.pull_request.user.login }}. Thanks for your PR.
+
+            After inspecting your changes someone with write access to this repo needs
+            to approve and run the workflow with the following slash_command.
+
+            * `/approve-test-run sha=<short commit sha to test>`:            Run helm chart linting and testing workflow
```

Comment:
is this comment a suggestion?

Reply:
this whole greeting idea was just to make the whole thing clear and avoid having to search in weird places what to do in order to run the workflow, but we can also avoid it entirely

Reply:
For context, if no one runs the workflow with the `/approve-test-run` an approved PR could end up merged without the check, because by default it is skipped